Essentials/src/com/earth2me/essentials/commands/EssentialsLoopCommand.java

```diff
@@ -19,27 +19,31 @@ public EssentialsLoopCommand(String command) {
         super(command);
     }
 
-    protected void loopOfflinePlayers(final Server server, final CommandSource sender, final boolean multipleStringMatches, boolean matchWildcards, final String searchTerm, final String[] commandArgs) throws PlayerNotFoundException, NotEnoughArgumentsException, PlayerExemptException, ChargeException, MaxMoneyException {
+    protected void loopOfflinePlayers(final Server server, final CommandSource sender, final boolean multipleStringMatches, boolean matchWildcards, final String searchTerm, final String[] commandArgs) throws Exception {
+        loopOfflinePlayersConsumer(server, sender, multipleStringMatches, matchWildcards, searchTerm, user -> updatePlayer(server, sender, user, commandArgs));
+    }
+
+    protected void loopOfflinePlayersConsumer(final Server server, final CommandSource sender, final boolean multipleStringMatches, boolean matchWildcards, final String searchTerm, UserConsumer userConsumer) throws Exception {
```

Comment:
This is an overloaded method, why change the name?

Reply:
Some uses of loop methods provide null to the args array which would create an ambiguous method call due to the fact that the `String[] args` argument isn't in `loopOfflinePlayersConsumer`